src/routes/AgreementEditRoute.js

```diff
@@ -35,6 +35,14 @@ class AgreementCreateRoute extends React.Component {
       path: 'erm/refdataValues/InternalContact/role',
       shouldRefresh: () => false,
     },
+    interfaces: {
```

Comment:
[Why are we fetching interfaces here?](https://issues.folio.org/browse/ERM-282?focusedCommentId=54363&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-54363)